### DISCUSSION TITLE
Default voice after startup

### DISCUSSION THREAD

**Discussion by rohoroho** [probonopd/MiniDexed#413] [Q&A] [answered]
Hi,

Is there a way to define a dedicated voice, when the RPI powers up?

I mean, when I have no rotary switch, can I define which voice will be the default? 

I like for example the piano voice as the default.

Thanks

**Comment by Banana71** [comment, marked as answer]
Hi,
sure, 
put the SD-Card in a PC and copy any Performance.ini (for example: 0000001_Piano2.ini) from the Performance-Folder to root and rename to performance.ini

**Reply by rohoroho** [reply]
Great! Thanks it worked fine!

**Comment by donluca** [comment]
Is there an option to have Minidexed remember the last patch used and load that by default when turned on?

**Reply by Banana71** [reply]
Hi, you can save the changing.

From the Wiki - Menue

Performance

Load 
Loads a saved performance file wither from performance.ini or from the performance directory. The flag "[L]" at the upper left of the display indicate the currently loaded performance file
Default Loads performance.ini
[Other entries] Load the respective file from the performance directory

Save 
Saves the performance configuration
Overwrite Writes changes to the currently loaded performance file, overwriting it
New Create a new performance file. By default a suggested name will be generated, but it can be edited before the new file will be created. To do that, Push select (= down) button and move rotary encoder or increment/decrement (= left/right) buttons to move cursor. Select (= down) to confirm and back (= up) to abort

Set as default 
Saves the current performance parameters to the default performance.ini file, overwriting it. This performance will be loaded each time the device is turned on

Delete 
Deletes the currently loaded performance file. Note that the performance.ini default performance file cannot be deleted

**Reply by donluca** [reply]
Thanks! I wasn't thinking of doing this via performances, but it makes perfect sense even for single voices without effects.
I just need to remember to save the performance before turning it off! I can already envision myself forgetting several times! 🤣
